daemon/daemon.go

```diff
@@ -589,13 +589,13 @@ func NewDaemon(c *Config) (*Daemon, error) {
 	if c.IsK8sEnabled() {
 		d.k8sClient, err = k8s.CreateClient(c.K8sEndpoint, c.K8sCfgPath)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("k8s create client: %s", err)
```

Comment:
I think this is still cryptic, let's be obvious: `"Unable to create k8s client: %s", err`